app/views/videos/show/_contributors_frame.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="panel panel-default module">
+  <div class="panel-heading">
+  	<b>Contributors</b>
+  	<a class="pull-right" href="#"><i class="fa fa-plus-circle"></i></a>
+  </div>
+  <% if @video.translators.empty? then %>
```

Comment:
If you're passing arguments into the partial, it's counter-intuitive to use the instance method `@video`. Just directly reference `video`.